package.json

```diff
@@ -1089,6 +1090,7 @@
     "async-mutex": "^0.5.0",
     "d3": "^7.9.0",
     "dataclass": "^3.0.0-beta.1",
+    "deep-equal": "^2.2.3",
```

Comment:
Do we still need this now that `created_at` is removed from the error events?

Reply:
I can remove the compare against the prior received message, sure. It *seems* that sidecar, somewhere in its byzantine connection state event emitting, will only emit if something about the connection has actually changed according to `.equals()` over the Connection object. Which should be equivalent to this deep-equals.

Removing ....

Reply:
Removed.